node/src/gql/query/mentor-client-data.ts

```diff
@@ -125,11 +126,20 @@ export const mentorData = {
         { mentor: null },
       ],
     });
-    const answers = await AnswerModel.find({
+    let answers = await AnswerModel.find({
       mentor: mentor._id,
-      status: Status.COMPLETE,
       question: { $in: questions.map((q) => q.id) },
     });
+    answers = answers.filter(
+      (a) =>
+        a.status === Status.COMPLETE ||
```

Comment:
this logic chain is used twice in this file and once in Mentor.ts, would be nice to have it moved to a helper function called something like "isAnswerComplete" (unless there's slight differences that I'm missing), ditto for "isAnswerIncomplete", just so we can have a single place to be more succinct about what a complete or incomplete answer is